### PR TITLE
W-10233797 add-chart

### PR DESCRIPTION
**Pull request by distroysf**
## Description
Write here a short description

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no
```
CTC Output:
Case approved, merging is possible.
The case is available at: https://gus.lightning.force.com/lightning/r/Case/500EE00000HKc5mYAD/view
Updated at Sun Dec 12 10:25:14 UTC 2021
```

### CODE REVIEW

File: src/stories/Chart.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  },
+  {
+    date: 'Jul, 2020',
+    ['Amazon']: 349420,
+    ['Brick and Morter']: 432400,
+    ['Commerce Cloud']: 210420,
+  },
+];
+
+const MyTooltip = ({
```
Comment (kaufguy) [inline src/stories/Chart.stories.tsx:64]: does it worth adding this tooltip as default to the component?

File: src/components/Chart.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Line as RechartsLine,
+  XAxis as RechartsXAxis,
+  YAxis as RechartsYAxis,
+  CartesianGrid as RechartsCartesianGrid,
+  Tooltip as RechartsTooltip,
+  ResponsiveContainer,
+} from 'recharts';
+import { useTheme } from '../hooks/theme.hooks';
+
+export const LineChart = RechartsLineChart;
```
Comment (kaufguy) [inline src/components/Chart.tsx:13]: I would suggest calling the component rechartsWrapper or rechartsChart and export * from 'recharts'

File: src/stories/Chart.stories.tsx (hunk 1 above)
Comment (distroysf) [reply src/stories/Chart.stories.tsx:69]: recharts have a default tooltip
this is an example to override it

**Review by kaufguy — APPROVED** [review]

**Pull request merged**